### DISCUSSION TITLE
Plasma stability

### DISCUSSION THREAD

**Discussion by JennyLeeeee** [Open-Systems-Pharmacology/Forum#1060] [Questions & Problems]
I want to input information on the PKsim & Mobi regarding plasma stability. So I need to implement that metabolism occurs in plasma. But I'm not sure about that information. There seems to be no implementation method in PKsim's Metabolism tab or transport & excretion. Can you do this with PKsim & mobi?

**Comment by tobiasK2001** [comment]
Dear Jenny,

that should be possible in PK-Sim as well. Just create and enzyme called e.g."PlasmaMetabolism" define that it is only expressed in plasma by setting relative expression in Plasma to 1 and in all other organs to zero.
Add in you compound BB a metabolism process that you can connect to your enzyme and than you should be ready to go.

Best, Tobias

**Reply by JennyLeeeee** [reply]
Dear Sir,

Thank you so much for your explanation. I'm sorry, but I have one more question.

What I have is data on rate constant (k) and half-life for in-vitro experiments. But when I open the metabolism tab of the compound, I don't know how to implement this.

When entering data related to in vitro, enzyme concetration is required. I think there is no way to put plasma metabolism because there is no enzyme and it proceeds by itself.

Is there a way to put only the data I have?
![metabolism](https://user-images.githubusercontent.com/96514837/147062272-360a43e2-a462-4474-a4c2-240250c0987e.png)

**Comment by tobiasK2001** [comment]
Hi,
in that case I would fit the process e.g. the Specific clearance to plasma concentration time profile of your compound in a middle out way. That would kind of ignore your in vitro data and you would use the presence of plasma metabolism as assumption for the reason for the fast disappearance of your compound.
As a quick thought, if you believe your in vitro data are predicting the in vivo situation I think your rate constant is most similar to the Specific clearance. As the compound is doing it by itself you can use you compound concentration in plasma as "enzyme concentration" start value. But that would be a simplification. Otherwise probaly you really might have to go to Mobi and include a decay reaction for your compound. But maybe there are more ideas from others...
